app/src/main/java/richard/chard/lu/android/areamapper/activities/AreaMapperActivity.java

```diff
@@ -624,9 +637,12 @@ protected void onDestroy() {
     }
 
     @Override
-    public void onImageSaved(String filePath) {
-        LOG.trace("Entry, filePath={}", filePath);
-        mapSnapshotPath = filePath;
+    public void onImageSaved(File imageFile) {
+        LOG.trace("Entry, imageFile={}", imageFile.getPath());
+
+        contentUri = FileProvider.getUriForFile(getApplicationContext(), IMAGE_PROVIDER_AUTHORITY, imageFile);
+
+        mapSnapshotPath = contentUri.toString();
```

Comment:
confirming that you are able to see this image back in CommCare in an image capture question ?

Reply:
yes, tested with Android 5.1, 7 and 11.

Reply:
great!